pkg/operator/loglevel/util.go

```diff
@@ -16,3 +24,56 @@ func LogLevelToKlog(logLevel operatorv1.LogLevel) int {
 		return 2
 	}
 }
+
+// CurrentLogLevel attempts to guess the current log level that is used by klog.
+// We can use flags here as well, but this is less ugly ano more programmatically correct than flags.
+func CurrentLogLevel() operatorv1.LogLevel {
+	switch {
+	case klog.V(8) == true:
+		return operatorv1.TraceAll
+	case klog.V(6) == true:
+		return operatorv1.Trace
+	case klog.V(4) == true:
+		return operatorv1.Debug
+	case klog.V(2) == true:
+		return operatorv1.Normal
+	default:
```

Comment:
I'm not sure this is reachable at all, given how `V(x)` performs its checks. Basically every level up to 2 (inclusively) will pick the previous case.

Reply:
i think this means we won't allow V(1) or V(0) and defer to V(2) always.

Reply:
We do allow both, it's just everything <2 is Normal.

Reply:
@soltysh we don't allow that for operatorv1.LogLevel is what I meant (there is no string value representing those, so if the current is 0 or 1, we report `Normal` (and won't change it).

Reply:
:+1: